src/snmp.c

```diff
@@ -1106,6 +1106,14 @@ static int csnmp_strvbcopy (char *dst, /* {{{ */
     src = (char *) vb->val.string;
   else if (vb->type == ASN_BIT_STR)
     src = (char *) vb->val.bitstring;
+  else if (vb->type == ASN_IPADDRESS)
+  {
+    return snprintf(dst, dst_size, "%d.%d.%d.%d",
```

Comment:
Please use `ssnprintf()` (note the extra "s"). The difference is that `ssnprintf()` guarantees null termination, `snprintf()` does not.